sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -2,34 +2,59 @@
   <student-page-container>
     <template #header>
       <header-navigator
-        title="Application details"
-        subTitle="Notice of Assessment"
+        title="View assessment"
+        subTitle="Assessment"
         :routeLocation="{
-          name: StudentRoutesConst.STUDENT_APPLICATION_DETAILS,
+          name: StudentRoutesConst.ASSESSMENT_AWARD_VIEW,
           params: {
-            id: applicationId,
+            applicationId: applicationId,
+            assessmentId: assessmentId,
           },
         }"
-      />
+        ><template #buttons v-if="!viewOnly">
+          <v-row class="p-0 m-0">
+            <v-btn
+              color="primary"
+              variant="outlined"
+              data-cy="cancelApplication"
+              @click="confirmCancelApplication"
+              >Cancel application</v-btn
+            ><v-btn
+              class="ml-2"
+              color="primary"
+              data-cy="AcceptAssessment"
+              @click="confirmAssessment()"
+              >Accept assessment</v-btn
+            >
+          </v-row>
+        </template>
+      </header-navigator>
     </template>
-    <notice-of-assessment-form-view :assessmentId="assessmentId" />
-    <v-row class="justify-center mt-4">
-      <v-btn color="primary" @click="confirmAssessment()">
-        Confirmation of assessment
-      </v-btn>
-    </v-row>
+    <notice-of-assessment-form-view
+      :assessmentId="assessmentId"
+      :view-only="viewOnly"
+      @assessmentData="assessmentData"
```

Comment:
I would suggest renaming it to indicate an event like `assessmentDataLoaded` or `dataLoaded` or `dataChanged`.
I would also recommend that the event argument can be the entire data that was loaded.